Form-Controls/styles.css

```diff
@@ -0,0 +1,127 @@
+:root {
+    --white:#FFF;
+    --black:#000;
+    --db:#E6DACD;
+    --lb:#F4ECE6;
+    --rb:#0150FD;
+    --header-font: 'Poppins', sans-serif;
+    --gen-font: 'Quicksand', sans-serif;
+}
+
+body {
+    margin: 0;
+}
+
+header {
+    background: var(--white);
+    height: 100px;
+}
+
+header div {
+    text-align: center;
+    padding-top: 1rem;
+}
+
+h1 {
+    font-family: var(--header-font);
+    margin: 0;
+}
+
+section {
+    background: var(--db);
+    height: 1000px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+}
+
+.section__container {
+    display: flex;
+    justify-content: center;
+    height: 800px;
+    width: 60%;
+    background: var(--white);
+    box-shadow: -15px 15px 15px grey;
+}
+
+form {
+    width: 80%;
+    padding-top: 3rem;
+}
+
+form input {
+    border: transparent;
+    border-bottom: 2px solid;
+    height: 40px;
+}
+
+form select {
+    height: 40px;
+}
+
+form label {
+    font-family: var(--header-font);
```

Comment:
This is a great example of why using clearer variable names improves your design. If this was your headings font, this variable would make sense here, but as it's your header font, I would expect it to only apply in the header.

Reply:
Yep, my bad. Should have been --heading-font instead of "header". Fixed